go.mod

```diff
@@ -7,14 +7,32 @@ module github.com/trustbloc/agent-sdk
 go 1.15
 
 require (
+	github.com/VictoriaMetrics/fastcache v1.5.8 // indirect
+	github.com/btcsuite/btcd v0.21.0-beta // indirect
+	github.com/cenkalti/backoff/v4 v4.1.0 // indirect
+	github.com/golang/protobuf v1.5.1 // indirect
 	github.com/google/uuid v1.2.0
 	github.com/gorilla/mux v1.8.0
-	github.com/hyperledger/aries-framework-go v0.1.7-0.20210324213044-074644c18933
+	github.com/hyperledger/aries-framework-go v0.1.7-0.20210326171010-c7ce51b1d6cb
 	github.com/hyperledger/aries-framework-go-ext/component/vdr/sidetree v0.0.0-20210306194409-6e4c5d622fbc
 	github.com/hyperledger/aries-framework-go-ext/component/vdr/trustbloc v0.0.0-20210325134531-84a30b2ecacb
-	github.com/hyperledger/aries-framework-go/component/storage/indexeddb v0.0.0-20210310140909-2ae2d7df101e
-	github.com/hyperledger/aries-framework-go/spi v0.0.0-20210320144851-40976de98ccf
+	github.com/hyperledger/aries-framework-go/component/storage/indexeddb v0.0.0-20210326171010-c7ce51b1d6cb
+	github.com/hyperledger/aries-framework-go/spi v0.0.0-20210326171010-c7ce51b1d6cb
 	github.com/igor-pavlenko/httpsignatures-go v0.0.23
+	github.com/jinzhu/copier v0.2.8 // indirect
+	github.com/kawamuray/jsonpath v0.0.0-20210127151053-2ab0d7f0a6ad // indirect
+	github.com/kilic/bls12-381 v0.1.0 // indirect
```

Comment:
this not good we need v0.0.0-20201104214312-31de2a204df8

Reply:
reverted